README.md

```diff
@@ -15,9 +15,10 @@ This will configure the target `census-eq` to point to the eq team on Concourse.
 ## Development Environments
 
 #### Prerequisites
-You must have a project with a *Google Container Registry (GCR)* already set up to be able to push built images.
-Set your registry in *variables.yaml* to `eu.gcr.io/<gcp_project_id>`. The `gcp_project_id` should be the ID of the GCP project under which the container registry is hosted.
-Ensure in the GCR settings, the visibility is set to Public. To do this, you must first initialize a registry manually via the UI or run the docker build step which would fail the first time since it would be private.
+- An existing GCP project.
+- *Google Container Registry (GCR)* enabled for the GCP project.
+- Set the registry in *variables.yaml* to `eu.gcr.io/<gcp_project_id>`. The `gcp_project_id` should be the ID of the GCP project under which the container registry is hosted.
```

Comment:
First one gives me the impression that this is talking about the current project. The second gives me the impression that it should just be any id. If that makes sense?.

Reply:
How about:

```md
#### Prerequisites
- An existing GCP project to provision the infrastructure in.
- *Google Container Registry (GCR)* enabled for a GCP project.
- Set the registry in *variables.yaml* to `eu.gcr.io/<gcp_project_id>`. `gcp_project_id` should be the ID of the GCP project under which the container registry is hosted.
- Ensure the visibility of the GCR is set to Public. To do this, you must first initialize a registry manually via the UI.
```